store/go_cache.go

```diff
@@ -132,11 +132,12 @@ func (s *GoCacheStore) Invalidate(ctx context.Context, options InvalidateOptions
 				cacheKeys = bytes
 			}
 
-			s.mu.RLock()
+			s.mu.Lock()
 			for cacheKey := range cacheKeys {
```

Comment:
Hello, @eko , I'm getting "concurrent map iteration and map write" error at this line in production. Have no clue why, could you please look, what I'm missing?

Reply:
Forget it, I had two instances of `GoCacheStore` by mistake.

Reply:
Alright, this is cool that you could be able to find your issue :)